### DISCUSSION TITLE
ELIGIBLE ENTRY / genuine material for submition

### DISCUSSION THREAD

**Discussion by erincon01** [microsoft/RAG_Hack#81] [Q&A] [answered]
hello.

for our video recording we want to show a small fragment of a video in youtube. 10 secs maximum.

is that ok with the rules? Thanks.

"
5. ELIGIBLE ENTRY

To be eligible, an entry must meet the following content/technical requirements:

Your entry must be your own original work; and Please note the video portion of your entry must be solely your own work, including but not limited to, the actual filming, editing, graphic design, etc. of the video.
Your entry cannot have been selected as a winner in any other contest; and
You must have obtained all consents, approvals, or licenses required for you to submit your entry; and
To the extent that entry requires the submission of user-generated content such as software, photos, videos, music, artwork, essays, etc., entrants warrant that their entry is their original work, has not been copied from others without permission or apparent rights, and does not violate the privacy, intellectual property rights, or other rights of any other person or entity. You may include Microsoft trademarks, logos, and designs, for which Microsoft grants you a limited license to use for the sole purposes of submitting an entry into this Contest; and

"

**Comment by multispark** [comment, marked as answer]
As long as you are not claiming another's work as your own (it is being used to support your project explanation), that is fine!
